docs/source/teams/installation.rst

```diff
@@ -72,11 +69,24 @@ That's it! Any operations you perform will be stored in a centralized location
 and will be available to all users with access to the same datasets in the
 Teams App or their Python workflows.
 
+
 .. _teams-cloud-credentials:
 
-Cloud credentials
+Cloud credentials and configuration
```

Comment:
nit: suggest not changing the section title since folks *might* be linking directly to the prior heading in the docs

Reply:
reverted back to original section title